Sources/Conduit/Auth/Cryptography/Errors/CryptoError.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CryptoError.swift
+//  Conduit
+//
+//  Created by John Hammerlund on 12/10/19.
+//
+
+import Foundation
+
+/// An error that occurs during a crypto operation
+public struct CryptoError: LocalizedError, Hashable {
+    /// The CryptoError code
+    public struct Code: Hashable {
+        public let rawValue: Int
+
+        public init(rawValue: Int) {
+            self.rawValue = rawValue
+        }
+    }
```

Comment:
What is the reasoning for making `Code` a struct instead of an enum with `Int` raw value?

Reply:
Enums are finite, and there are a lot of things that could go wrong when dealing with crypto operations. I imagined the list could grow, which would introduce breaking changes as an enum.